Pair_Up/ClearPiece.h

```diff
@@ -0,0 +1,6 @@
+#ifndef CLEARPIECE_H
+#define CLEARPIECE_H
+#include "Map.h"
+Map clearPiece(Map temporary);//����Map�࣬������Ҫ��ȥ��Map��
```

Comment:
@jushe 
I think it's unnecessary to return the `Map` object, return the `maplists` only is enough.
And by the way, I need you to separate the `check & judge` parts as a **independent function**,which will use to **judge whether the map is dead**.